### PR TITLE
LPS-165423 | Add tests tests ErrorAppearsWhenHTMLBlank, HTMLElementConatinsInvlaidCharacters, HTMLElementMustHaveOneHyphen, HTMLElementMustStartWithLowercase and path REMOTE_APPS_ERROR_WITH_MESSAGE 

### PR DESCRIPTION
**Pull request by bronsonsoda**
**Background**
Make a tests that ensures an error appears when the HTML is left blank.

**Test Plan**
_ErrorAppearsWhenHTMLBlank_
Given Add Custom Element Remote Apps admin page
When required fields Name,HTML Element Name, and JS URL are blank
And When attempt to save remote app
Then error message "HTML element name is empty"
And Then Remote App is not added

https://issues.liferay.com/browse/LPS-165425

**Background**
Make a tests that ensures invalid characters throw an error when entered.

**Test Plan**
_HTMLElementConatinsInvlaidCharacters_
Given Add Custom Element Remote Apps
When type "vanilla counter" in JS URL field
And When attempt to save the remote app
Then  error message 'HTML Element name contains invalid character " "' appears under HTML Element field

https://issues.liferay.com/browse/LPS-165745

**Background**
Make a tests that ensures an error is thrown if HTML element starts with an uppercase letter.

**Test Plan**
_HTMLElementMustStartWithLowercase_
Given Add Custom Element Remote Apps
When type "Vanilla counter" in HTML Element Name field
And When attempt to save the remote app
Then error message "HTML element name must start with a lowercase letter" appears under HTML Element field

https://issues.liferay.com/browse/LPS-165834

**Background**
Make a tests that ensures an error is thrown if HTML element doesn't contain a hyphen.

**Test Plan**
_HTMLElementMustHaveOneHyphen_
Given Add Custom Element Remote Apps
When type "vanilla.counter" in HTML Element Name field
And When attempt to save the remote app
Then error message "HTML element name must contain at least one hyphen" appears under HTML Element field

https://issues.liferay.com/browse/LPS-165835

### CODE REVIEW

**Comment by bronsonsoda** [comment]
ci:test:frontend-remote-apps:nocompile

**Comment by bronsonsoda** [comment]
ci:test:frontend-remote-apps:nocompile

**Comment by bronsonsoda** [comment]
@john-co should be good to go, local tests are on the tickets

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/remotewebapps/RemoteApps.testcase [hunk 1]
```diff
@@ -656,6 +684,81 @@ definition {
...
+		}
+
+		task ("When type 'vanilla counter' in HTML Element field and When attempt to save the remote app") {
+			Type(
+				key_text = "HTML Element Name",
+				locator1 = "TextInput#ANY",
+				value1 = "vanilla counter");
+
+			Button.clickPublish();
+		}
```
Comment (john-co) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/remotewebapps/RemoteApps.testcase:703]: separate into two tasks

**Comment by bronsonsoda** [comment]
ci:forward

**Comment by bronsonsoda** [comment]
what the...

**Comment by bronsonsoda** [comment]
hmmmmm

**Comment by bronsonsoda** [comment]
ci:forward

**Comment by bronsonsoda** [comment]
ci:test:sf

**Comment by bronsonsoda** [comment]
ci:test:sf

**Comment by bronsonsoda** [comment]
ci:forward

**Pull request closed without merging**